.github/workflows/release.yml

```diff
@@ -8,9 +8,9 @@ jobs:
     name: PyPI Release
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
     - name: PyPI Upload
-      uses: FeatureLabs/gh-action-pypi-upload@master
+      uses: FeatureLabs/gh-action-pypi-upload@main
```

Comment:
Main Branch added:
https://github.com/FeatureLabs/gh-action-pypi-upload/tree/main

Reply:
Perhaps we should switch to using tagged releases of this action so that the branch name isn't even a factor